gallery/src/pageConfig.ts

```diff
@@ -86,6 +87,7 @@ const pageConfig: PageConfig = {
     NxFontAwesomeIcon: NxFontAwesomeIconPage,
     NxVulnerabilityDetails: NxVulnerabilityDetailsPage,
     NxSubmitMask: NxSubmitMaskPage,
+    NxTablePage: NxTableComponentPage,
```

Comment:
```suggestion
    NxTable: NxTableComponentPage,
```

Reply:
I did a bit of renaming to match the naming style we used for NxTextInput now, see if you like it better.